src/main/kotlin/io/easybreezy/user/model/User.kt

```diff
@@ -97,7 +116,7 @@ class User private constructor(id: EntityID<UUID>) : AggregateRoot<UUID>(id) {
 }
 
 enum class Status {
-    ACTIVE, WAIT_CONFIRM
+    CREATED, WAIT_CONFIRM, ACTIVE
```

Comment:
Maybe it will be a better set a pending status?

Reply:
It's good idea, but i think "pending" status assumes to have another status like "denied" or "refused". And we'll have to make user refusal.

Reply:
This is a good idea)  to add refused or deleted or archived status.

Reply:
> archived

 sounds nice